test/validation/api/pktio/lso.c

```diff
@@ -689,9 +689,9 @@ static void lso_send_custom_eth(const uint8_t *test_packet, uint32_t pkt_len, ui
 
 	ODPH_DBG("\n    Sent payload length:     %u bytes\n", sent_payload);
 
-	/* Wait 1 sec to receive all created segments. Timeout and MAX_NUM_SEG values should be
+	/* Wait 0.1 sec to receive all created segments. Timeout and MAX_NUM_SEG values should be
```

Comment:
For better maintainability, perhaps the actual time could be dropped from the comment.